src/services/ledger/volume.ts

```diff
@@ -22,6 +22,11 @@ export const TxnGroups = {
     LedgerTransactionType.OnchainIntraLedger,
     LedgerTransactionType.LnIntraLedger,
   ],
+  tradeIntraAccountTxBaseVolumeSince: [
+    LedgerTransactionType.WalletIdTradeIntraAccount,
+    LedgerTransactionType.OnChainTradeIntraAccount,
```

Comment:
I guess this is a type for future use?

Reply:
Done: https://github.com/GaloyMoney/galoy/pull/1682/commits/ca1f2e0d401ebddf53bd7394d67d2038584f3c87

Reply:
Yup, we'll have the `OnChainPaymentFlowBuilder` soon so I figured we could add this in here as prep for that.